log4j-nosql/src/main/java/org/apache/logging/log4j/nosql/appender/mongodb/MongoDbProvider.java

```diff
@@ -53,18 +53,22 @@
     private final DB database;
     private final String description;
     private final WriteConcern writeConcern;
+    private final Boolean isCapped;
```

Comment:
Could you stick to the primitive types instead? Unless these are nullable, but it doesn't appear so.